client/src/components/form/NewAppVersionForm.jsx

```diff
@@ -15,7 +15,7 @@ const DHISVersions = config.ui.dhisVersions
 
 const validate = values => {
     const errors = {}
-    const requiredFields = ['version', 'file']
+    const requiredFields = ['version', 'file', 'channel', 'minVer', 'maxVer']
```

Comment:
Questions:

- Is channel hardcoded to "stable" in the client?
- Are minVer and maxVer not possible to send empty strings? so e.g. minVer of 2.33 without a maxVer is not valid?

Reply:
Available channels are loaded dynamically from the backend.

This validation just adds up to be in line with the same requirements as when creating a new app.

Regarding max version maybe that requires a discussion how we want to handle it before implementing. What I can tell from the API we allow empty max but the database doesn't. 

I wanted to bring the applications validation to a consistent working state before touching that area.

Reply:
OK, if there's a plan to account for it I'm OK with this.